source/_patterns/02-molecules/branding/logo.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="150" height="200" viewBox="0 0 1 1">
```

Comment:
See above comment about duplicate logo.

Reply:
Oh! It wasn't really a duplicate logo. They were separate logos to solve using a graybox for PL vs. the "real" one for Drupal. Probably should simply set the site logo and let it be determined by the file?